include/sysdolphin/baselib/fog.h

```diff
@@ -3,6 +3,53 @@
 
 #include <dolphin/types.h>
 
-void HSD_FogSet(s32);
+#include <sysdolphin/baselib/object.h>
+
+typedef struct _HSD_FogAdj {
+    HSD_Obj parent;
+    s16 center; // 0x08
```

Comment:
Correct these comments to the `/* 0xoffset */ type var;` style, please.

Reply:
This will need to be addressed in a new issue because both styles are being used throughout the codebase right now. Your style is probably more readable but this style is more friendly to intellisense (particularly VS Code's).

Reply:
The reason it's mixed usage is because of me copy-pasting from structs in FRAY, which I think was done here. It's been something I've tried to cleanup as I actually decompile files themselves.

I was using this style because of Intellisense at the time and changed to match how other projects handle it with `/* 0xA */, but we should settle on which needs to be done if that's the case.

Reply:
Tracked in #557.